### ISSUE TITLE
Vapor auto exit after benchmark

### ISSUE THREAD

**Issue by wildfiremedia** [qutheory/vapor#355]
On Macbook Pro 10.11
vapor run --release

Once I passed ```wrk -t1 -c1000 -d30s http://127.0.0.1:8080``` it will either to manage 1000 connections or automatically terminate itself with the errors message:
```
[1465048982] [INFO] GET /

Error: Could not run project.
Note: Make sure you are using Swift 3.0 Preview 1
```

**Comment by LoganWright** [comment]
@wildfiremedia can you pull the `socks` branch and check if you can reproduce? We're working out some similar issues at the moment. I'm unable to repro on that branch.

**Comment by wildfiremedia** [comment]
How do I pull socks branch with vapor? Would it be easier to have vapor cli to switch different branch instead of downloading new code from github?

**Comment by tannernelson** [comment]
This has been fixed in 0.11
